lexicons/com/atproto/server/updateEmail.json

```diff
@@ -0,0 +1,29 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.updateEmail",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Update an account's email.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["email"],
+          "properties": {
+            "email": { "type": "string" },
+            "token": {
+              "type": "string",
+              "description": " Requires a token from com.atproto.sever.requestEmailUpdate if the account's email has been confirmed."
```

Comment:
I wonder if we'll run into issues here when a user loses access to their former, confirmed email address.  It's not uncommon when work-specific or school-specific email addresses are used to create accounts, and then the user leaves their job or school.

Just to toss out one unflesh-out alternative—the user could receive a token by providing their credentials (e.g. username and password, or in the future 2fa).  This is similar to how e.g. github has you enter "sudo mode" before adding a new email address or performing other sensitive actions.

Reply:
I think they'll need to go through an out-of-band support flow then

I like the alternative 👍 I don't see the current appraoch as being backwards incompatible with it tho. May be something we do when we get in 2fa.